tasks/deploy.yml

```diff
@@ -13,4 +16,6 @@
 - include: launch_cloudformation.yml
   when: not wimpy_needs_elb
 
-- include: scaling_policies.yml
\ No newline at end of file
+- include: scaling_policies.yml
+
+- include: notify.yml
```

Comment:
What do you mean? The task inside `notify.yml` won't be executed if the github project is not passed as parameter